resources/views/pages-legacy/gameInfo.blade.php

```diff
@@ -541,7 +541,7 @@ function resize() {
                     $interestedUsers = UserGameListEntry::where('type', UserGameListType::Develop)
                         ->where('GameID', $gameID)
                         ->count();
-                    echo "<div><a class='btn btn-link' href='" . route('game.dev-interest', $gameID) . "'>View Developer Interest ($interestedUsers)</a></div>";
+                    echo "<div><a class='btn btn-link' href='" . route('game.dev-interest', ['game' => $gameID]) . "'>View Developer Interest ($interestedUsers)</a></div>";
```

Comment:
There must still be a broken route somewhere on this page. I get the "`Argument #2 ($arguments) must be of type array, int given`" error trying to view a game.

Reply:
Found it - fixed in latest.